### PR TITLE
Add sales filter to filtering and columns

### PR DESCRIPTION
**Pull request by MelvinAchterhuis**
### 1. Why is this change necessary?

Adds a new filter for merchants so they can easily filter their best selling products

### 2. What does this change do, exactly?

Adds a new filter to the product filtering

### 3. Describe each step to reproduce the issue or behaviour.

N/A

### 4. Please link to the relevant issues (if any).

N/A

### 5. Checklist

- [x] I have rebased my changes to remove merge conflicts
- [ ] I have written tests and verified that they fail without my change
- [x] I have created a [changelog file](https://github.com/shopware/platform/blob/trunk/adr/workflow/2020-08-03-implement-New-Changelog.md) with all necessary information about my changes
- [x] I have written or adjusted the documentation according to my changes
- [x] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

<a href="https://gitpod.io/#https://github.com/shopware/platform/pull/2828"></a>

### CODE REVIEW

File: src/Administration/Resources/app/administration/src/module/sw-product/snippet/de-DE.json [hunk 1]
```diff
@@ -500,6 +500,9 @@
         "label": "Kategorien",
         "placeholder": "Kategorien auswählen ..."
       },
+      "salesFilter": {
+        "label": "Verkauf"
```
Comment (tinect) [inline src/Administration/Resources/app/administration/src/module/sw-product/snippet/de-DE.json]:
```suggestion
        "label": "Verkäufe"
```

**Review by tinect — CHANGES_REQUESTED** [review]

**Comment by J-Rahe** [comment]
Hey @MelvinAchterhuis,
your PR looks good so far, but could you add the option to also display the sales in the listing (same as with currencies)?

**Comment by MelvinAchterhuis** [comment]
> Hey @MelvinAchterhuis, your PR looks good so far, but could you add the option to also display the sales in the listing (same as with currencies)?

@J-Rahe , how would you see this working with parent products that has no sales? A sum of al variant sales? (seems like me this could have an impact on the performance when the parent has a lot of variants)

**Comment by J-Rahe** [comment]
Hey @MelvinAchterhuis, sorry that it took so long. I have now confirmed with the PO that:
1. The main listing should have the option to show the sales and just show an placeholder(e.g. "-") for variant products.
2. The variant listing (from the main listing and in the product tab) should have the same option to enable sales being shown, and will then show the sales for each variant.

**Comment by MelvinAchterhuis** [comment]
@J-Rahe ,
I've updated the PR

**Comment by keulinho** [comment]
Thanks again for your contribution, your PR got finally merged! :tada: :blue_heart:

**Pull request closed without merging**